scripts/copyhooks.js

```diff
@@ -0,0 +1,11 @@
+const fs = require('fs'); 
+
+const srcDir = './hooks/';
+const dstDir = './.git/hooks/'
+
+fs.readdirSync(srcDir).forEach(file => {
+  srcFile = srcDir + "/" + file;
+  dstFile = dstDir + "/" + file;
+  fs.createReadStream(srcFile).pipe(fs.createWriteStream(dstFile));
+  fs.chmodSync(srcFile , '755');
+});
```

Comment:
Refer the `path` core module here for path related operations...
forward slash won't work for windows etc.

`path.resolve` etc...

Reply:
done

Reply:
can be resolved